Search_Engine/Search_Engine.csproj

```diff
@@ -0,0 +1,45 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyVersion>7.0.0.0</AssemblyVersion>
+    <Description>https://github.com/BHoM/AGS_Toolkit</Description>
+    <Version>5.0.0</Version>
+    <Authors>BHoM</Authors>
+    <Copyright>Copyright © https://github.com/BHoM</Copyright>
+    <RootNamespace>BH.Engine.Search</RootNamespace>
+    <FileVersion>7.1.0.0</FileVersion>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
Is this required? Just had a look at other projects and it only seems to have:
  `<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
    <OutputPath>..\Build\</OutputPath>
  </PropertyGroup>`

Reply:
I think it is but I'll double check and get back to you

Reply:
As discussed on Teams, this isn't actually needed because of the way our installers build, and was me having an out-of-memory issue with my head making me think we needed it 😄